package.json

```diff
@@ -129,6 +130,7 @@
     "jsonwebtoken": "^8.5.1",
     "jwk-to-pem": "^2.0.4",
```

Comment:
what about jszip? it's required in the code but I don't see it here...

Reply:
hmm maybe the latest merge removed it, will check